src/notifications/screens/notifications.screen.js

```diff
@@ -163,6 +167,8 @@ class Notifications extends Component {
     } = this.props;
     const { type } = this.state;
 
+    this.props.getNotificationsCount();
```

Comment:
Could we have this as part of our destructured object up above:

```
     const {
       getUnreadNotifications,
       getParticipatingNotifications,
       getAllNotifications,
       getNotificationsCount
      } = this.props;
```

So we can just call `getNotificationsCount()` here